.github/workflows/forest.yml

```diff
@@ -0,0 +1,193 @@
+name: Integration tests
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - '*.md'
+      - '*.rb'
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '*.md'
+      - '*.rb'
+  schedule:
+    - cron: "0 0 * * *"
+
+env:
+  CI: 1
+  CARGO_INCREMENTAL: 1
```

Comment:
`sccache` cannot cache incremental builds. This value should be `0`.

Reply:
So we had a bug? I mean, the value was there in the `rust.yml`.

Reply:
Yeah, it has been a problem for a while.

Reply:
Only a slight problem, though. It increases the miss rate of `sccache` by a few percent.